FrontEnd/src/App.js

```diff
@@ -30,6 +33,7 @@ function App() {
           <Route path="/SignUpPage" element={<SignUpPage />} />
           <Route path="/PrivacyPolicyPage" element={<PrivacyPolicy />} />
           <Route path="/TermsAndConditions" element={<TermsAndConditions />} />
+          <Route path="/Search" element={<Search />} />
```

Comment:
why do all path names start from upper case? Check the [url naming best practices ](https://www.mtrmarketing.com/blog/url-naming-best-practices)

Reply:
It's better to change all uppercase URLs and separate them by dash. Example `privacy-policy` instead of `PrivacyPolicyPage`

Reply:
Done